### DISCUSSION TITLE
Future plans?

### DISCUSSION THREAD

**Discussion by Frans-L** [akveo/react-native-ui-kitten#1335] [General]
## 💬 Question

Hey! I really love this library. Really polished and well documented 💯 

However, I'm wondering what the future plans of the UI-Kitten. It seems like there haven't been any updates since May 2020.

Is this library still being maintained and developed further? 😄

**Comment by Bandwagoner** [comment]
Also interested in knowing this considering I started a new project with this library. It's been great aside from a few small bugs.

**Comment by artyorsh** [comment]
Hi everyone 👋  

We postponed the maintenance of the library and related products ([Eva](https://github.com/eva-design/eva), [Kitten Tricks](https://github.com/akveo/kittenTricks)) since we had to focus on other products which actually bring money to support our Open Source. That was for the COVID-19 reasons. We'll definitely get back when the things become better with huge amount of fresh ideas, packages to extend the library and many more :)

Currently I spend my personal time to support several open issues, but I'm not responsible to merge PR's, add Bug Fixes without other teammates review and approval.

Sorry for not putting this info for a such long period and thanks for your interest. Stay tuned :)

**Comment by artyorsh** [comment]
@Bandwagoner if you have a project where you would like to use the library but a few issues is a blocker, I would suggest starting with a fork of the project, where you can close them. If you have questions regarding this, please [open an issue](https://github.com/akveo/react-native-ui-kitten/issues/new/choose) so I can help with that. 
Thanks for your understanding.

**Comment by robertwt7** [comment]
The library is really good compared to the other ui kit out there! Hopefully you guys can get back on your feet post covid :), all the best.

I'll keep using the library in the meanwhile

**Comment by anthowm** [comment]
@artyorsh There is the possibility to activate github sponsor and openCollective to try to speed up the return to the repository (like chakraUI). I think that like me other developers can help the project in this way.

**Comment by murbanowicz** [comment]
@artyorsh Would you try to discuss with your teammates even one-off session of merging PRs and releasing one new version? Is it possible at all?

EDIT: Nebular is getting release after release, so why the same is not happening here? :( 
UI Kitten is one of the best UI libraries for RN and it is very sad that the maintainer is not willing to push it forward while there is so much input from the community (PRs).

Can you please reconsider what to do with this project at Akveo? 

I am pretty sure there is lot of people in the community here willing to help if there is no funds at Akveo to push the development of this library.

**Comment by vitorverasm** [comment]
We ❤️ this UI library, I've used a lot and it surprised me how well documented and consistent this components are. We understand that for now(for COVID-19 purposes) the maintenance of this project is a little slow. But there's something that we can try to reach like:
* Make a 2021 roadmap
* Review open PR's an put them into milestones
* Open sponsor options for this project

**Comment by kuasha420** [comment]
I'm also a long time user and would love to help in any way possible.

**Comment by Dammic** [comment]
As this have been moved to discussions, and the last update message was in October, I'd like to ask - how are things going now?

Could you give us an update on that @artyorsh ? How are things looking in the short term future and long-term? I would be very grateful for your response!

We don't have any blockers in our team currently (if we have, we'll make our own fork), but we are considering if sticking to the library is a good business idea if the support is going to be completely dropped.

**Reply by artyorsh** [reply]
Well, as you might noticed, we started working slowly with fixing existing issues. The goal is to reduce the amount of open issues as much as possible. So you may expect several patch releases in the near future.

**Comment by Nasseratic** [comment]
Greetings,

Since the last reply on this was 8 months ago, and the last actual released version was 6 months ago.

@artyorsh
We got a great UI library that does not deserve to die! Please let us know the current status. Is there are any plans for supporting the library in the near future?

Our team is trying to take a decision on using UI kitten or not, and not knowing if their maintenance will be dropped or not is a very important factor for the decision.

**Comment by malashkevich** [comment]
Hello @Nasseratic,
It's nice to know that you are interested in UI Kitten. 
Let me explain the current status of it.

We created UI Kitten as a base UI framework for [our company, Akveo](https://www.akveo.com/). All mobile projects that we are working on at Akveo are using UI Kitten. All new projects that are starting are also started with UI Kitten. So I can assure you that we will support UI Kitten at least for our internal needs.

But it is worth mentioning that we don't have at the moment a dedicated developer who is working full time on this. So issues could wait longer than previously. We are working on resolving this, but it's not that easy also from the financial standpoint :)

So we don't have a plan of sunsetting the UI Kitten, instead, we would like to update Eva and UI KItten to a new version, introducing new components. But again ETA for this is a quite difficult topic.

Please let me know if you have any questions.
